.github/workflows/createchangelog.yml

```diff
@@ -111,9 +111,10 @@ jobs:
         id: "commit-template"
         with:
           commit_message: "Upload Changelogs for Release ${{ inputs.tag }}"
-          commit_author: nomi-ceu-management[bot] <155502507+nomi-ceu-management[bot]@users.noreply.github.com>
-          commit_user_name: nomi-ceu-management[bot]
-          commit_user_email: 155502507+nomi-ceu-management[bot]@users.noreply.github.com
+          commit_author: horde-of-greg-management[bot] <155502507+horde-of-greg-management[bot]@users.noreply.github.com>
```

Comment:
Nomi CEu management is a dedicated github app they made, so changing this to a bot / account that doesn't exist might lead to unexpected behavior
We could make a github app ourselves, or simplify the changelog process to not require uploading it as a commit to the repo and add it to github releases and cf instead

This change is present in multiple places, so be careful when resolving this

Reply:
yeah i know, its why i added the comment. Idk how we should fix it, but if you have a fix feel free to propose it, whatever works, works.

Reply:
@Mireole can you check the latest commit, pretty sure just changing the bot ID fixes it. Also please go check the bot permissions in the settings of our org